templates/server/src/main/resources/archetype-resources/core/src/main/java/__packageInPathFormat__/general/common/impl/config/ApplicationObjectMapperFactory.java

```diff
@@ -18,7 +23,11 @@
   public ApplicationObjectMapperFactory() {
 
     super();
-    // register polymorphic mapping here - see https://github.com/oasp-forge/oasp4j-wiki/wiki/guide-json#json-and-inheritance
-    getExtensionModule().addAbstractTypeMapping(CsrfToken.class, CsrfTokenImpl.class);
+    // see https://github.com/oasp-forge/oasp4j-wiki/wiki/guide-json#json-and-inheritance
```

Comment:
should be https://github.com/devonfw-wiki/devon4j/wiki/guide-json#json-and-inheritance